libcob/fileio.c

```diff
@@ -3345,37 +3345,27 @@ void
 cob_file_save_status (cob_file *f, cob_field *fnstatus, const int status)
 {
 	int	k, indent = 15;
+	int	skip_exn = 0;
 
+	/* TODO: internally let status be an enum (also in internal storage);
+	   and then map here to an i-o status "per dialect",
+	   inluding alphanumeric 0x and 9/123 status values */
```

Comment:
That's definitive a 4.0 TODO, also because there are some MF-specific extended codes in 4.x already.

Reply:
Added to TODO list.